presto-base-jdbc/src/main/java/com/facebook/presto/plugin/jdbc/JdbcErrorCode.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.plugin.jdbc;
+
+import com.facebook.presto.spi.ErrorCode;
+import com.facebook.presto.spi.ErrorCodeSupplier;
+
+public enum JdbcErrorCode
+        implements ErrorCodeSupplier
+{
+    // Connectors can use error codes starting at EXTERNAL
+
+    /**
+     * A requested data conversion is not supported.
```

Comment:
A `SQLException` while writing could mean anything. So I think the error code is correct, but the comment is not.

However, there might be specific SQLState codes that could be given a more specific error code.